bt_sim:
  gui: True
  seed: 12

active_grasp:
  frame_id: task
  length: 0.3
  base_frame_id: panda_link0
  ee_frame_id: panda_hand
  ee_grasp_offset: [0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.065]
  camera:
    frame_id: cam_optical_frame
    info_topic: /cam/depth/camera_info
    depth_topic: /cam/depth/image_raw

vgn:
  model: $(find vgn)/assets/models/vgn_conv.pth
  finger_depth: 0.05
